modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -523,6 +533,66 @@ protected function register_controls() {
 		] );
 
 		$this->end_controls_section();
+
+		$this->start_controls_section( 'section_box_style', [
+			'label' => esc_html__( 'Box', 'elementor' ),
+			'tab' => Controls_Manager::TAB_STYLE,
+		] );
+
+		$this->add_group_control(
+			Group_Control_Background::get_type(),
+			[
+				'name' => 'box_background_color',
+				'types' => [ 'classic', 'gradient' ],
+				'exclude' => [ 'image' ],
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Border::get_type(),
+			[
+				'name' => 'box_border',
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_responsive_control(
```

Comment:
I tried to test the border radius control and it didn't work for me.
Can you double check it?

Reply:
Test: https://drive.google.com/file/d/1taXlHw33fbol_0HQrVeZhi_9cIpItTVa/view

It seems to work fine on my side.

Reply:
Re-tested, works.